aten/src/ATen/templates/TensorBody.h

```diff
@@ -910,6 +910,11 @@ Tensor make_tensor(Args&&... args) {
   return Tensor(c10::make_intrusive<T>(std::forward<Args>(args)...));
 }
 
+template <typename T, typename... Args>
+Tensor make_view_tensor(Args&&... args) {
+  return Tensor(c10::make_intrusive<T>(std::forward<Args>(args)..., /*is_view*/true));
+}
```

Comment:
Nyeh, I hate this. You can still use `make_tensor` to create a view tensor, and this constructor breaks symmetry with `make_unique` which this function is modeled off of. Remember the enum constructor trick from DispatchKeySet? You can use that here too.